pypit/core/arskysub.py

```diff
@@ -212,6 +208,29 @@ def skyoptimal(wave,data,ivar, oprof, sortpix, sigrej = 3.0, npoly = 1, spatial
     return (sky_bmodel, obj_bmodel, outmask)
 
 
+def order_pixels(pixlocn, lord, rord):
```

Comment:
This is kind of the same thing as `arpixels.slit_pixels` (and the algorithm that it calls: `locate_order`). I don't think we need this in the skysub routines as well (but I might be mistaken).

Reply:
Hmm... @profxj may need to handle this one.  Will keep it for now.

Reply:
keep it